### PR TITLE
redesign of filter companies and config-fix

### PR DESCRIPTION
**Pull request by jonatan-kruse**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
```
Comment (jonatan-kruse) [inline package.json]: Nice! this is way better!

File: app.config.ts [hunk 2]
```diff
@@ -41,10 +41,9 @@ export default {
   extra: {
     isDev: process.env.NODE_ENV === 'development',
     isProd: process.env.NODE_ENV !== 'development',
-    //backendUrl: process.env.NODE_ENV === 'development' ? (process.env.BACKEND_URL ? process.env.BACKEND_URL : 'https://nexpo-dev.arkadtlth.se/api') : 'https://nexpo.arkadtlth.se/api',  
-    backendUrl: process.env.NODE_ENV === 'development' ? (process.env.BACKEND_URL ? process.env.BACKEND_URL : 'http://localhost/api') : 'http://localhost/api',  },
-    eas : {
-      projectId: "736e5c8b-3245-43fc-9803-293e803f584e"
```
Comment (elnaseyer) [inline app.config.ts:47]: Varför vill vi ta bort eas?

File: package.json [hunk 3]
```diff
@@ -2,6 +2,7 @@
   "main": "node_modules/expo/AppEntry.js",
   "scripts": {
     "start": "expo start",
+    "tunnel": "expo start --tunnel",
```
Comment (jonatan-kruse) [inline package.json:5]: Veri nice

**Comment by jonatan-kruse** [comment]
https://user-images.githubusercontent.com/42937653/197348118-515aeba0-eb22-44a1-b36a-09e17acff88a.mp4
Redid the filtering of companies. Needs to be tried on IOS before merging

**Comment by jonatan-kruse** [comment]
Actually looks very nice now imo, please check how it looks on ios and then we can merge it!

**Review by elnaseyer — APPROVED** [review]

**Pull request merged**